src/DssSpell.sol

```diff
@@ -20,66 +20,81 @@ pragma solidity 0.6.12;
 
-    // Math
     uint256 constant MILLION = 10 ** 6;
     uint256 constant WAD     = 10 ** 18;
+    uint256 constant RAY     = 10 ** 27;
 
-    // Turn on office hours
-    function officeHours() public override returns (bool) {
-        return true;
-    }
+    address constant CES_WALLET  = 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da;
+    address constant RISK_WALLET = 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c;
 
-    function actions() public override {
-        address MCD_CLIP_USDT_A = DssExecLib.getChangelogAddress("MCD_CLIP_USDT_A");
-        address MCD_VEST_DAI    = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
+    address constant MCD_VEST_MKR_TREASURY = 0x6D635c8d08a1eA2F1687a5E46b666949c977B7dd;
 
-        // Offboard USDT-A
-        // https://vote.makerdao.com/polling/QmRNwrTy?network=mainnet#vote-breakdown
+    uint256 constant APR_01_2021 = 1617235200;
 
-        // 15 thousand DAI maximum liquidation amount
-        DssExecLib.setIlkMaxLiquidationAmount("USDT-A", 15_000);
+    uint256 constant CURRENT_BAT_MAT          = 150 * RAY / 100;
+    uint256 constant CURRENT_LRC_MAT          = 175 * RAY / 100;
+    uint256 constant CURRENT_ZRX_MAT          = 175 * RAY / 100;
+    uint256 constant CURRENT_UNIV2AAVEETH_MAT = 165 * RAY / 100;
+    uint256 constant CURRENT_UNIV2LINKETH_MAT = 165 * RAY / 100;
 
-        // flip breaker to enable liquidations
-        DssExecLib.setValue(MCD_CLIP_USDT_A, "stopped", 0);
+    uint256 constant TARGET_BAT_MAT          = 800 * RAY / 100;
+    uint256 constant TARGET_LRC_MAT          = 2600 * RAY / 100;
+    uint256 constant TARGET_ZRX_MAT          = 900 * RAY / 100;
+    uint256 constant TARGET_UNIV2AAVEETH_MAT = 400 * RAY / 100;
+    uint256 constant TARGET_UNIV2LINKETH_MAT = 300 * RAY / 100;
 
-        // authorize breaker
-        DssExecLib.authorize(MCD_CLIP_USDT_A, DssExecLib.clipperMom());
-
-        // set liquidation ratio to 300%
-        DssExecLib.setIlkLiquidationRatio("USDT-A", 30000);
+    function add(uint x, uint y) internal pure returns (uint z) {
+        require((z = x + y) >= x, "ds-math-add-overflow");
+    }
+    function sub(uint x, uint y) internal pure returns (uint z) {
+        require((z = x - y) <= x, "ds-math-sub-underflow");
+    }
 
-        // remove liquidation penalty
-        DssExecLib.setIlkLiquidationPenalty("USDT-A", 0);
+    // Turn on office hours
+    function officeHours() public override returns (bool) {
+        return true;
+    }
 
+    function actions() public override {
+       
+        //
+        // Direct payment
+        //
+        
+        // CES-001 - 1_223_552 DAI - 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da
+        // https://vote.makerdao.com/polling/QmbM8u7Q?network=mainnet#vote-breakdown
+        DssExecLib.sendPaymentFromSurplusBuffer(CES_WALLET, 1_223_552);
+
+        //
+        // MKR vesting
+        //
+
+        TokenLike(DssExecLib.getChangelogAddress("MCD_GOV")).approve(MCD_VEST_MKR_TREASURY, 700 * WAD);
```

Comment:
I think this is probably fine for the initial deployment this week if we consider it a soft launch, but it will be tough to track individual approval amounts like this in the future, we'll probably just want to pick a larger number next time.

Reply:
I think if we pair these changes anytime we increase or decrease, we should be okay.  It feels like we need two new DssExecLib functions though.

Reply:
`increaseApproval` and `decreaseApproval`, I'll add a card.